app/_locales/en/messages.json

```diff
@@ -3166,6 +3172,15 @@
   "tokenDecimalFetchFailed": {
     "message": "Token decimal required."
   },
+  "tokenDecimalTitle": {
```

Comment:
A few comments here

1. What's the different between `tokenDetails` and `tokenDetailsTitle`. They seem to show the same message (only capitalisation changed)

2. Do we want to stick with one case type e.g Token Decimal or Token decimal.

Reply:
@segun Yes, we want to stick with one case type, so I deleted `tokenDetailsTitle` and use just `tokenDetails` to display. You can see changed here: a28aee5e9014c2d1872c083c5e1692440209d540